src/index.html

```diff
@@ -1,41 +1,32 @@
 <!doctype html>
-<html lang="en" ng-app="app">
+<html lang="en">
 
 <head>
+  <base href="/">
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Kashti</title>
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-  <title>
-    Kashti | {{ $state.current.name | capitalize }}
-  </title>
-
-  <base href="/">
-
-  <link href="assets/css/app.css" rel="stylesheet" type="text/css">
   <link rel="icon" type="image/png" href="assets/images/favicon.png">
   <link rel="icon" type="image/x-icon" href="assets/images/kashti-icon.png">
   <link rel="apple-touch-icon-precomposed" href="assets/images/kashti-icon.png">
 
 </head>
 
-<body zf-close-all zf-esc-close>
+<body zf-close-all zf-esc-close ng-app="app" ng-strict-di>
```

Comment:
This means that the application will fail to invoke functions which do not use explicit function annotation (and are thus unsuitable for minification). This is good as the Angular app will fail to start if code hasn't explicitly had dependencies injected.